### DISCUSSION TITLE
Support of @Index order option

### DISCUSSION THREAD

**Discussion by darky** [mikro-orm/mikro-orm#4854] [General]
**Is your feature request related to a problem? Please describe.**
Now it's impossible to set order of your index via `@Index` decorator
For example, in Postgres, default order for index is `ASC`
But very often, need to setup `DESC` order for index 

**Describe the solution you'd like**
`order` option for `@Index` decorator, which receiving literals `ASC` or `DESC`

**Comment by B4nan** [comment]
You can use index expressions to define any kind of index

https://mikro-orm.io/docs/defining-entities#indexes

**Reply by picardplaisimond** [reply]
I know this is already possible with expressions, but it would be a really nice to just have an option to define the sorting order to keep the code cleaner (instead of writing the whole raw SQL string). The sort order of indexes matters (especially for compound indexes). It's not an urgent feature, but it should be on the roadmap. I feel like it would be really useful to a lot of people.
